src/components/Select/common/InternalSelect.tsx

```diff
@@ -250,9 +250,9 @@ export const InternalSelect = ({
       if (e.key === "Enter") {
         e.preventDefault();
         if (highlighted) {
-          onSelect(highlighted);
+          onSelect(highlighted, "default", e);
```

Comment:
this is a breaking change, the 2nd argument is optional, and it changed from `undefined` to `"default"`
this might be correct, but it's still a breaking change
Not sure how to handle it. Pass undefined here instead?

Reply:
We probably don't care about breaking changes though, with us being the only consumers, and versioning not even having a minor part 🤔

Reply:
I think passing `undefined` is the correct move.